applications/Cyborg.md

```diff
@@ -0,0 +1,178 @@
+
+### Work Verification System
+
+For this grant application, we are developing a decentralized task verification mechanism to incorporate an on-chain validation system for containerized tasks executed in an off-chain environment. The main components will include the Cyborg AppChain, which will be deployed on Tanssi's Dancebox testnet, the client interface (K3s worker system), an Oracle, and a testing suite.
+
+<img alt="Screenshot 2024-02-15 at 1 09 56 PM" src="https://github.com/Cyborg-Network/Grants-Program/assets/93442895/add65d92-0bf6-40df-a3fb-715fc0efce54">
+
+The objective is to upload a Docker config file along with a publicly available dataset such as IPFS and deploy the container over the IPFS dataset inside one of the nodes of the K3s cluster. The executed computation result will be pushed to the Oracle, which will then feed it to the Cyborg substrate chain. Subsequently, the verification pallet will initiate task execution on another connected cluster to reverify the result. If the initial result and verified result are the same, the chain will approve the computation and add it to the block.
+
+### Ecosystem Fit
+
+Cyborg Network augments the Polkadot ecosystem by ushering in decentralized edge computing, crucial for the growing demand of AI and IoT applications. By primarily catering to Web2 application providers, we're paving the way for a fresh wave of users to engage with the Polkadot ecosystem, fueling its expansion.
+
+Though Cyborg and the [Phala Network](https://phala.network/) share aspirations of decentralized computing within the Polkadot/Kusama environment, our technical paths and business aspirations are distinct. Phala Network provides off-chain compute infrastructure for smart contract-based applications by enabling users to integrate 'phat' contracts with their existing smart contract logic. While Cyborg is focused on creating utility for web2 applications.
+
+Another related project appears to be [Wetee](https://github.com/wetee-dao), which employs a comparable technical architecture. However, Cyborg leverages software-level privacy mechanisms such as Full Homomorphic Encryption (FHE), providing us with the capability to deploy containerized instances even on non-TEE CPU architectures. We are developing products for a validated market that has already attracted interest from various AI applications and 50 small data center providers to join the network once it goes live.
+
+![Write here](https://github.com/Cyborg-Network/Grants-Program/assets/93442895/a09109d1-e8f7-4ae5-8f4c-2eafc4b6ef6a)
+
+Our ultimate goal is to establish a decentralized edge computing network capable of executing targeted low-latency tasks to enhance system performance and efficiency. This approach aims to reduce cloud expenses without necessarily replacing cloud services, thereby opening up new markets, particularly in real-time computation for edge AI and edge IoT devices.
```

Comment:
No, these are mostly edge data centers that focus on providing low-latency compute by strategically locating the infrastructure closer to where it's mostly concentrated. The cost reduction comes into play because of the overall reduction of cloud footprint of the system and not by offering cloud services at cheaper rates.

We conducted a pilot analysis with one of the interested companies, an AI company that installs security cameras with computer vision-based surveillance abilities. They had a requirement to shift some of their data point analysis tasks to the edge and also wanted the edge servers to perform compression on the generated video footage before pushing it into their upstream S3 bucket in AWS. This process reduces their overall cloud footprint by up to 66%, which in turn translates into reduced expenditure on cloud services. The overall savings, as per the analogy, was up to 55%, including the cost of our services.